src/javascript/app_2/Modules/Trading/Components/Form/TradeParams/start_date.jsx

```diff
@@ -42,7 +42,7 @@ const StartDate = ({
         >
             <Dropdown
                 name='start_date'
-                value={start_date}
+                value={parseInt(start_date)}
```

Comment:
@easteregg 
What is your idea if we use the syntactic sugar `+start_data` instead of parseInt?

Reply:
I prefer `parseInt` it covers more cases and it is more strict comparing to unary plus operator.